src/github.com/getlantern/golog/golog.go

```diff
@@ -257,35 +258,33 @@ func (l *logger) errorSkipFrames(arg interface{}, skipFrames int) error {
 	default:
 		err = fmt.Errorf("%v", e)
 	}
-	buf := bufferPool.Get()
-	defer bufferPool.Put(buf)
-	l.print(GetOutputs().ErrorOut, buf, skipFrames+4, "ERROR", err)
-	return report(err, buf.String())
+	linePrefix := l.print(GetOutputs().ErrorOut, skipFrames+4, "ERROR", err)
```

Comment:
With this change, there's no longer a need to pass the whole formatted message, which is nice because the reporter doesn't have to do as much parsing anymore.